supportedBackends/google/pipelines/v2beta/src/main/scala/cromwell/backend/google/pipelines/v2beta/api/request/GetRequestHandler.scala

```diff
@@ -164,4 +166,26 @@ trait GetRequestHandler { this: RequestHandler =>
 
     starterEvent.toList ++ filteredExecutionEvents ++ completionEvent
   }
+
+  // Future enhancement: parse as `com.google.api.services.lifesciences.v2beta.model.DelayedEvent` instead of
+  // generic `Event` and take advantage of `getMetrics` which has a string array of problem quota(s):
+  //   "metrics": [
+  //     "CPUS"
+  //   ]
+  private def isQuotaDelayed(events: List[Event]): Boolean = {
+    events.sortBy(_.getTimestamp).reverse.headOption match {
+      case Some(event) =>
+        quotaMessages.exists(event.getDescription.contains)
```

Comment:
This is shorthand for

`quotaMessages.exists(message => event.getDescription.contains(message))`?

Reply:
Yes, or the intermediate level of shortening
```
quotaMessages.exists(event.getDescription.contains(_))
```